generate.js

```diff
@@ -341,40 +350,62 @@ function downloadFromUrl(srcUrl, appendUrl, apiCache, cacheKey, onEachComplete,
         });
     });
 }
-/////////////////////////////////// Major step 3 - Generate the indicated ouptut files ///////////////////////////////////
-function generateApis(buildIdentifier, targetOutputPathList, buildFlags, apiSrcDescription) {
-    console.log("Generating PlayFab APIs from specs: " + apiSrcDescription);
+function generateApis(buildIdentifier, target) {
+    console.log("Generating PlayFab APIs from specs: " + sdkGeneratorGlobals.apiSrcDescription);
+    var genConfig = null;
+    var genConfigPath = path.resolve(target.destPath, "genConfig.json");
+    try {
+        genConfig = require(genConfigPath);
+        console.log("Loaded genConfig at: " + genConfigPath);
+    }
+    catch (_) {
```

Comment:
>_ [](start = 11, length = 1)

Clarification: is this naming the err variable as _ ? or is this a special JavaScript feature?

Reply:
In JS, I believe it allocates a variable named "_" and assigns it to the value of the caught exception.
By convention in some scripting languages, a variable "_" is just an extremely temporary trash variable. Multiple scripting languages use it as convention, and one uses it as explicit "ignore this var" and it optimizes out that assignment. (But I can't remember which is which)